### PR TITLE
Improvements to "Reset the Arduino bootloader on the Nano ESP32" [HC-1612]

### PR DESCRIPTION
**Pull request by seaxwi**
Document bootloader mode option and clarify some steps based on article feedback

### CODE REVIEW

File: content/Hardware Support/Nano Family/Reset-the-Arduino-bootloader-on-the-Nano-ESP32.md [hunk 1]
```diff
@@ -41,4 +70,20 @@ Follow these steps:
...
+
+### Troubleshooting
+
+#### dfu-util: No DFU capable USB device available
+
+If you see this error, it means that the sketch is still being uploading with the regular upload process.
+
+Try this:
+
+* Ensure that the **Esptool** option is selected in the Sketch > Upload Using Programmer menu. It needs to be actively selected, even if no other options are available. You will know the Esptool option is selected if a checkmark is displayed next to the name.
```
Comment (Renat0Ribeir0) [inline content/Hardware Support/Nano Family/Reset-the-Arduino-bootloader-on-the-Nano-ESP32.md:83]:
```suggestion
* Ensure that the **Esptool** option is selected in the **Sketch > Upload Using Programmer** menu. It needs to be actively selected, even if no other options are available. You will know the Esptool option is selected if a checkmark is displayed next to the name.
```

File: content/Hardware Support/Nano Family/Reset-the-Arduino-bootloader-on-the-Nano-ESP32.md [hunk 2]
```diff
@@ -10,29 +10,58 @@ These are some reasons you may want to reset the bootloader:
...
 4. Open Arduino IDE.
 
 5. Go to **Tools > Port** and select the board (it may be identified as an arbitrary ESP32 board).
 
 6. Go to **Tools > Board** and select **Arduino ESP32 Boards > Arduino Nano ESP32** (or **esp32 > Arduino Nano ESP32** if you're using the full esp32 package).
 
-7. Open **Tools > Programmer** and ensure **Esptool** is selected.
+7. Open the **Tools > Programmer** menu.
+
+8. Click on and the **Esptool** option to select it.
```
Comment (Renat0Ribeir0) [inline content/Hardware Support/Nano Family/Reset-the-Arduino-bootloader-on-the-Nano-ESP32.md:62]:
```suggestion
8. Click on the **Esptool** option to select it.
```
Typo

**Review by Renat0Ribeir0 — CHANGES_REQUESTED** [review]
Looks good! Left some minor suggestions. Also, since it is mentioned that this process works in Arduino IDE and Cloud Editor (_"3. Select Arduino Nano ESP32 in Arduino IDE or the Cloud Editor."_), the intro should be updated.

**Comment by seaxwi** [comment]
> since it is mentioned that this process works in Arduino IDE and Cloud Editor (_"3. Select Arduino Nano ESP32 in Arduino IDE or the Cloud Editor."_), the intro should be updated.

@Renat0Ribeir0 Good point! Programmer selection is not available in the Cloud Editor, but that's only needed for the option 2. I have made some adjustmens in e568360e980ffbcb632cc38096186c9cc09476c0

**Review by Renat0Ribeir0 — APPROVED** [review]

**Pull request merged**